CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
-## 3.0.0-rc1 - 2021-12-03
+## 3.0.0 - 2021-12-06
+
+### Added
+- Look up host for 'terminus drush' / 'terminus wp' commands via an alternate nameserver, if selected (#2253)
+- Typehint param and return value in lookupHostViaAlternateNameserver (#2253)
+- D9ify command (see docs)
```

Comment:
D9ify was actually broken out into the [terminus conversion tools plugin](https://github.com/pantheon-systems/terminus-conversion-tools-plugin), so I think this should probably be removed.

Reply:
Removed in b64067b.